paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/AppearanceBottomSheetDialogFragment.kt

```diff
@@ -940,26 +941,26 @@ private fun RowStyleDropDown(style: String, rowStyleSelectedCallback: (String) -
             DropdownMenuItem(
```

Comment:
Can we change this to be a for loop instead? There's a lot of duplication here.

Reply:
Updated 👍